packages/format-url/index.ts

```diff
@@ -0,0 +1,19 @@
+export default function formatUrl(
```

Comment:
Having format-url as an independent package feels like an overkill. Let's try to aggregate things a bit more

Reply:
this is complete.  I removed the format-url package and people can use the formatUrl export of the common package